migrations/1607009558538-create-notifications-table.js

```diff
@@ -0,0 +1,40 @@
+var db = require('./db')
+
+function singleQuotify(item) {
+  return "'" + item + "'"
+}
+
+var types = [
+  'highValueTransaction',
+  'fiatBalance',
+  'cryptoBalance',
+  'compliance',
+  'error'
+]
+  .map(singleQuotify)
+  .join(',')
+
+exports.up = function (next) {
+  const sql = [
+    `
+    CREATE TYPE notification_type AS ENUM ${'(' + types + ')'};
+    CREATE TABLE IF NOT EXISTS "notifications" (
```

Comment:
remove the "if not exists", we need to fail hard if the database is in a bad state.